pkg/apis/operator/v1alpha1/tektonconfig_defaults.go

```diff
@@ -94,4 +94,24 @@ func (tc *TektonConfig) SetDefaults(ctx context.Context) {
 			tc.Spec.Pruner.Resources[index] = value
 		}
 	}
+
+	// chains defaults
+	if tc.Spec.Chain.ArtifactsTaskRunFormat == "" {
+		tc.Spec.Chain.ArtifactsTaskRunFormat = "in-toto"
+	}
+	if tc.Spec.Chain.ArtifactsTaskRunStorage == nil {
+		tc.Spec.Chain.ArtifactsTaskRunStorage = ptr.String("oci")
+	}
+	if tc.Spec.Chain.ArtifactsPipelineRunFormat == "" {
+		tc.Spec.Chain.ArtifactsPipelineRunFormat = "in-toto"
+	}
+	if tc.Spec.Chain.ArtifactsPipelineRunStorage == nil {
+		tc.Spec.Chain.ArtifactsPipelineRunStorage = ptr.String("oci")
+	}
+	if tc.Spec.Chain.ArtifactsOCIFormat == "" {
+		tc.Spec.Chain.ArtifactsOCIFormat = "simplesigning"
+	}
+	if tc.Spec.Chain.ArtifactsOCIStorage == nil {
+		tc.Spec.Chain.ArtifactsOCIStorage = ptr.String("oci")
+	}
```

Comment:
@jkhelil these defaulting will not impact existing chains CR, 
However, if user updates a field on a TektonConfig CR, not related to chains also changes the chains behavior.
Will that impact the existing chains instance?

Reply:
yes i guess, it is going to set defaults, but thats what we expect, we do want to set these defaults

Reply:
this defaulting should be done in chains CR, and from here we should be just calling that